iron-autogrow-textarea.html

```diff
@@ -69,7 +65,7 @@
 
     <!-- size the input/textarea with a div, because the textarea has intrinsic size in ff -->
     <div class="textarea-container fit">
-      <content id="textarea" select="textarea"></content>
+      <textarea id="textarea" required$="[[required]]" maxlength$="[[maxlength]]"></textarea>
```

Comment:
I will add the binding but I think the call to `_update()` at `attached()` takes care of this.